roles/scaffold_tests/files/inventory.yaml.j2

```diff
@@ -0,0 +1,11 @@
+all:
```

Comment:
We can have the inventory in `test/integration/` only as `targets` is a place just for the roles.
https://github.com/ansible/ansible/tree/devel/test/integration/